shared/js/background/utils.es6.js

```diff
@@ -313,6 +313,38 @@ function isSameTopLevelDomain (url1, url2) {
     return firstDomain === secondDomain
 }
 
+/**
+ * Checks the config to see if a feature is enabled. You can optionally pass a second "customState"
+ * parameter to check if the state is equeal to other states (i.e. state === 'beta').
+ *
+ * @param {String} featureName - the name of the feature
+ * @param {String} customState - An optional custom state to check for
+ * @returns {bool} - if feature is enabled/matches specified state
+ */
+function isFeatureEnabled (featureName, customState) {
```

Comment:
What is `customState` used for? I don't see it being used.

Reply:
It is future proofing. If we have a feature that uses a `beta` state for example we can check for it using `customState` instead of just checking if its equal to `enabled`.

Reply:
Should that not be checked inside the function rather than passed in?

Aka when the state is `beta` we check that the users channel is `beta` / they've enabled some custom mode somehow.

As we're not currently using this I think we should drop it; the format supports different states which is a great improvement.